### PR TITLE
Feature literature review

### PR DESCRIPTION
**Pull request by HenrikFrtz**
Added markdown file with literature links

### CODE REVIEW

File: literature/literature.MD [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+## Microservice architecture
+
+General information: [Microservice Patterns and Best Practices](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=5259455&query=microservice+python)
+
+[Python Microservices Development](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=6739164)
+: Take a look on `Chapter 1: Understanding Microservices`
+
+[Python Microservices Development](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=4998695)
```
Comment (philippabele) [inline literature/literature.MD:17]: This line seems to be a duplicate from line 14.

**Review by philippabele — CHANGES_REQUESTED** [review]

File: literature/literature.MD (hunk 1 above)
Comment (HenrikFrtz) [reply literature/literature.MD:17]: Actually this are two different books, but they deal both about Microservices in Python, to understand the basics of microservice architectures.

Comment (philippabele) [reply literature/literature.MD:17]: Do not reference to books in this way. Please provide information about the author, ISBN and so on.

**Comment by HenrikFrtz** [comment]
@philippabele I updated the information now. Are you missing s.th.?

**Review by HenrikFrtz — COMMENTED** [review]
Ok

**Review by HenrikFrtz — COMMENTED** [review]
ok

**Comment by HenrikFrtz** [comment]
I've changed the Literature.MD long ago. I'm struggeling how to approve your change request, because otherwise I cannot merge this branch. I'd honor if you could take a look.

**Comment by philippabele** [comment]
@HenrikFrtz , I would do it like this:
-copy your literature.md file in temporary directory
- create a new branch and create a new PR
- delete the former branch and PR

**Pull request closed without merging**